internal/resolvers/youtube/model.go

```diff
@@ -9,3 +9,15 @@ type youtubeTooltipData struct {
 	LikeCount    string
 	DislikeCount string
 }
+
+type youtubeChannelTooltipData struct {
+	Title        string
+	PublishDate  string
```

Comment:
This shouldn't be called `PublishDate` or anything near it. If it denotes the creation date of the youtube channel, label it as such.

Youtube themselves write it as "Joined Jan 28, 2009" - we could use that phrasing.

Reply:
Renamed to `JoinedDate`